### PR TITLE
Tag COBRA v0.1.0

### PR DESCRIPTION
**Pull request by laurentheirendt**
Repository: [COBRA.jl](https://github.com/opencobra/COBRA.jl)
Release: v0.1.0
Travis: [Build Status](https://travis-ci.org/opencobra/COBRA.jl)

**Changes:**

- adding functionality to find reactions (`findRxnIDs`) based on reaction IDs
- extended tests
- keyword arguments in `distributedFBA`
- `onlyFluxes`, `logFiles` and `saveChunks` arguments in `distributedFBA`
- improved tutorial and documentation
- compatibility for Julia `v0.6`

cc: @syarra

### CODE REVIEW

File: COBRA/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+MathProgBase 0.6.1
```
Comment (tkelman) [inline COBRA/versions/0.1.0/requires]: was there a bugfix or new function in 0.6.1 that wasn't in 0.6.0 that you're relying on?

Comment (laurentheirendt) [inline COBRA/versions/0.1.0/requires]: yes, [compatibility fixes](https://github.com/JuliaOpt/MathProgBase.jl/releases/tag/v0.6.1) for `Julia 0.6`.

Comment (tkelman) [inline COBRA/versions/0.1.0/requires]: but COBRA requires julia 0.5, not julia 0.6. MathProgBase 0.6.0 would work fine on julia 0.5

Comment (laurentheirendt) [inline COBRA/versions/0.1.0/requires]: For now, I want to support Julia `0.5` and `0.6`. 

Actually, in `MathProgBase 0.6.1`, [this commit](https://github.com/JuliaOpt/MathProgBase.jl/commit/c691ed772edb05f50756b3dd57f43728b4ba0b4d) broke my tests (no default solver anymore). I hence made it a minimum requirement to have `MathProgBase 0.6.1` and adapted all tests to take that fix into account.

**Pull request merged**